MSAL/src/cache/MSALTokenCacheAccessor.m

```diff
@@ -106,7 +106,6 @@ - (BOOL)saveRefreshToken:(MSALRefreshTokenCacheItem *)rtItem
 
     BOOL result = [_dataSource addOrUpdateRefreshTokenItem:rtItem context:ctx error:error];
 
-    [event setStatus:result ? MSAL_TELEMETRY_VALUE_SUCCEEDED : MSAL_TELEMETRY_VALUE_FAILED];
```

Comment:
Ashish requested to remove status properties (copy-paste from telemetry channel: can we remove msal.status from the token events, its an artifact of OneDrive, i don't think it provides value).